packages/generate-prism-languages/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "generate-prism-languages",
+  "private": true,
+  "scripts": {
+    "start": "ts-node ./index.ts"
```

Comment:
This is probably a nit: but "start" makes it seem like this is a long-running process. Something like "generate" might be more appropriate?